src/BizHawk.Client.Common/movie/MovieSession.cs

```diff
@@ -154,7 +154,8 @@ public bool HandleLoadState(TextReader reader)
 				}
 				else if (Movie.IsPlayingOrFinished())
 				{
-					LatchInputToLog();
+					int previousFrame = Movie.Emulator.Frame - 1;
+					Movie.Session.MovieController.SetFrom(Movie.GetInputState(previousFrame));

```

Comment:
Only thing I can think of is saving and loading after the end of a movie, does that break?

Reply:
That doesn't seem to be possible in the first place, so I assume that's safe.